sass/components/_wysiwyg.scss

```diff
@@ -5,4 +5,7 @@
   .ecl-file + article {
     margin-top: $ecl-spacing-2-xs;
   }
+  article + .ecl-file {
+    margin-top: $ecl-spacing-2-xs;
+  }
```

Comment:
Can we try with this below and see how it looks on all combinations?
```scss
.ecl-editor {
  article,
  .ecl-file {
    margin-top: $ecl-spacing-l;
    margin-bottom: $ecl-spacing-l;
  }
}
```

Reply:
The spacing works fine like this but previously when we fixed spacing in wysiwyg we used $ecl-spacing-2-xs as agreed with Tamas. I think $ecl-spacing-l is too much since it would be in one field and not between fields. It looks like this with $ecl-spacing-l:
<img width="1183" alt="spacing" src="https://user-images.githubusercontent.com/22908988/95315410-bdf30580-0892-11eb-9558-f360d689c822.png">

Which one should I keep?

Reply:
I'm using $ecl-spacing-2-xs for now.

Reply:
Ok sure, if agreed with @kugta  is ok for course!